arm-servicebus/2017-04-01/swagger/servicebus.json

```diff
@@ -0,0 +1,3152 @@
+        "x-ms-examples": {
+          "RulesListBySubscriptions": { "$ref": "../examples/Rules/RuleListBySubscription.json" }
+        },
+        "description": "List all the rules within given topic-subscription",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/namespaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/topicNameParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved list of Rules.",
+            "schema": {
+              "$ref": "#/definitions/RuleListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceBus/namespaces/{namespaceName}/topics/{topicName}/subscriptions/{subscriptionName}/rules/{ruleName}": {
+      "put": {
+        "tags": [
+          "Rules"
+        ],
+        "operationId": "Rules_CreateOrUpdate",
+        "x-ms-examples": {
+          "RulesCreateOrUpdate": { "$ref": "../examples/Rules/RuleUpdate.json" }
+        },
+        "description": "Creates a new rule",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/namespaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/topicNameParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ruleNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Rule"
+            },
+            "description": "Parameters supplied to create a rule."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Rule created.",
+            "schema": {
+              "$ref": "#/definitions/Rule"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Rules"
+        ],
+        "operationId": "Rules_Delete",
+        "x-ms-examples": {
+          "RulesDelete": { "$ref": "../examples/Rules/RuleDelete.json" }
+        },
+        "description": "Deletes an existing rule.",
```

Comment:
Please add description as they are vital part of the documentation for each service. 

There are `11` warning of  `DescriptionAndTitleMissing `. Please fix.

https://travis-ci.org/Azure/azure-rest-api-specs/jobs/243507256#L1291